model/task/db.go

```diff
@@ -813,6 +813,21 @@ func findAllTaskIDs(q db.Q) ([]string, error) {
 	return ids, nil
 }
 
+func FindStuckDispatching() ([]Task, error) {
+	tasks, err := FindAll(db.Query(bson.M{
+		StatusKey:       evergreen.TaskDispatched,
```

Comment:
Why would these tasks be set to dispatched?

Reply:
stu found a case where things were stuck in dispatching, and in the queue. it happens sometimes, but not often.